src/typescript/12-generics/generics.md

```diff
@@ -1,23 +1,23 @@
 @page learn-typescript/generics Generics
 @parent learn-typescript 12
+@outline 3
 
 @description Learn about what Generics are, why they are useful, and how to create a linked list using Generics in TypeScript.
 
 @body
 
-
 ## Overview
 
-In this part, we will:
+In this section, we will:
```

Comment:
“you will” https://bitovi.atlassian.net/wiki/spaces/BSG/pages/665976833/Bitovi+Academy+Style+Guide#Overview